plugin/teal.vim

```diff
@@ -0,0 +1,19 @@
+
+function s:LoadPlugins()
+	if exists("loaded_matchit")
+		let b:match_ignorecase = 0
+		let b:match_words=
+			\ '\<\%(do\|enum\|record\|function\|if\)\>:' .
+			\ '\<\%(return\|else\|elseif\)\>:' .
+			\ '\<end\>,' .
+			\ '\<repeat\>:\<until\>'
+	endif
+	if exists("loaded_endwise")
```

Comment:
Shouldn't that be `g:loaded_endwise`? `loaded_endwise` is not defined on my machine.

With that change, the plugin appears to work as expected.